### PR TITLE
Write forecast to pvsite database

### PR DESCRIPTION
**Pull request by devsjc**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -43,3 +44,14 @@ add-ignore = 'D200,D202,D210,D212,D415,D105'
 [tool.mypy]
 check_untyped_defs = true
 ignore_missing_imports = true
+
+[tool.ruff]
```
Comment (simlmx) [inline pyproject.toml:48]: Shouldn't `ruff` be added as a python dependency? Isn't it redundant with `pydocstyle, `flake8`, etc., which are still marked as dependencies?

File: configurations/aws_config.yaml [hunk 2]
```diff
@@ -9,3 +9,4 @@ nwp:
 pv_metadata_path: s3://pvsite-ml-models-development/meta_inferred.csv
 
 pv_db_url: ${OCF_PV_DB_URL}
+pvsite_db_url: ${OCF_PVSITE_DB_URL}
```
Comment (simlmx) [inline configurations/aws_config.yaml:12]: Aren't we writing in the same database as we are reading? Should those two URLs be merged into one?

File: pyproject.toml (hunk 1 above)
Comment (devsjc) [reply pyproject.toml:48]: It is indeed, I'll change that

**Comment by peterdudfield** [comment]
@devsjc any chance we can split these in 2? One PR for the write function, one for linting. That'll make it a bit easier to read. 
I'm sure @simlmx will be able to review it

**Comment by devsjc** [comment]
The linting was just because I kept getting caught out by the linting pipeline 😅 I can split them out though

**Comment by simlmx** [comment]
Superseded by #28

**Pull request closed without merging**